lib/edge_commander/commands/commands.ex

```diff
@@ -52,9 +53,11 @@ defmodule EdgeCommander.Commands do
   end
 
   def list_rules(user_id) do
-    Rule
-      |> where(user_id: ^user_id)
-      |> Repo.all
+    query = from r in Rule,
+      left_join: m in Member, on: r.user_id == m.user_id,
+      where: (m.member_id == ^user_id or r.user_id == ^user_id)
+    query
+    |>  Repo.all
```

Comment:
try to form this query more like Ecto piping method, instead of this way, to keep it same as we have done in all queries.